lib/transfer.c

```diff
@@ -601,7 +601,7 @@ static CURLcode readwrite_data(struct Curl_easy *data,
       nread = 0;
     }
 
-    if((k->bytecount == 0) && (k->writebytecount == 0)) {
+    if(!k->bytecount) {
```

Comment:
Can this be true multiple times though?

Reply:
I don't think it risks getting called more times now than it did before. But I also think that if it does, it might actually make it more accurate.